webhooks/2025-04-15.yml

```diff
@@ -2536,9 +2541,16 @@ components:
         payment_method:
           type: string
           description: Type of payment method (e.g., "card")
+          enum:
+            - link
+            - card
         status:
           type: string
-          description: Current status of the payment source (e.g., "active", "inactive", "expired")
+          description: Current status of the payment source
+          enum:
+            - active
```

Comment:
How is this being calculated currently? `payment_method.type.link` does not return a `status` attribute or `exp_month` and `exp_year` to utilise for determining if it has `expired`.

Reply:
I can't find it being returned anywhere, but in theory in the future it could be. We can compare the expiration date we store on the payment source with the current time, or we can update the status when Stripe returns an expired payment method error in response to attempting to charge the card.

Reply:
Sorry, the `status` is being returned, but the value `expired` isn't.

Reply:
Ah, got it. Apologies, I was referring to the `status` attribute, to understand what makes a `BillingPaymentSource` land in `inactive` or `expired` given that `payment_method.type.link` does not return `expired` from Stripe.

Reply:
When you delete the payment method from our system, it gets marked as deactivated, regardless of the type. But the documentation was wrong, that value is "disconnected", not "inactive", upon investigation. I'll get that fixed.